packages/front-end/services/metrics.tsx

```diff
@@ -115,7 +122,7 @@ export function formatPercent(
     maximumSignificantDigits: 3,
     ...options,
   });
-  return percentFormatter.format(value);
+  return percentFormatter.format(Math.round(value * 100000) / 100000);
```

Comment:
Another fun javascript trivia.  What is `Math.round(1.255*100)`?  Hint: It's not 126.

(I think in this particular case you're ok, but only because 100000 as the multiplier doesn't happen to have this bug)

Reply:
I wanted to use `roundingPriority` (https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/NumberFormat/NumberFormat#roundingpriority) but it seems like it wasn't available.